src/app/lib/views/player/player.js

```diff
@@ -343,8 +343,6 @@
 
         onShow: function () {
             $('#header').removeClass('header-shadow').hide();
-            // Test to make sure we have title
-            win.info('Watching:', this.model.get('title'));
```

Comment:
I don't see why exactly, we see the title on the app, the log doesnt add anything